### DISCUSSION TITLE
Dependency issue using pandas-ai

### DISCUSSION THREAD

**Discussion by manuelzo** [sinaptik-ai/pandas-ai#835] [Q&A]
Dear all, I'm trying to use pandas-ai in a LLM RAG tool called [Cheshire Cat](https://cheshirecat.ai) but unfortunately cheshire-cat 1.4.3 requires pydantic==2.4.2, but pandas-ai have pydantic 1.10.13 which is incompatible. I'm looking for a solution. Thanks in advance. M

**Comment by lordpba** [comment]
same issue here...
hmm... I think there is a problem with pydantic, in my case is in conflict with crewai library that requires an earlier version of pydantic
